packages/block-editor/src/components/media-placeholder/index.native.js

```diff
@@ -43,8 +43,12 @@ function MediaPlaceholder( props ) {
 		getStylesFromColorScheme,
 		multiple,
 		value = [],
+		children,
+		customStyles,
 	} = props;
 
+	const styles = customStyles === undefined ? defaultStyles : customStyles;
```

Comment:
What are the exact styles that we want to override? If it's just the height or something like that, what do you think if we just pass those instead of all the styles for the placeholder? e.g we could pass a prop `customContainerStyles` and we can merge it with `emptyStateContainerStyle`.